src/_guides/language/effective-dart/design_migrated.md

```diff
@@ -789,13 +790,14 @@ Sometimes an instance field doesn't change after it has been initialized, but
 can't be initialized until after the instance is constructed. For example, it
 may need to reference `this` or some other field on the instance. In cases like
 that, consider making the field `late final`. When you do, you may also be able
-to initialize the field at its declaration.
+to [initialize the field at its declaration][init at decl].
 
+[init at decl]: usage#do-initialize-fields-at-their-declaration-when-possible
```

Comment:
We always use the full path, even though I'm not sure it matters any more. (It used to.)

Btw, I plan to make a pass to fix all the _migrated URLs inside of files, so don't worry if you need to use the _migrated URL.

```suggestion
[init at decl]: /guides/language/effective-dart/usage#do-initialize-fields-at-their-declaration-when-possible
```